<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QWidget" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>461</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>16</height>
    </size>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="nor_setting">
    <attribute name="title">
     <string>常规设置</string>
    </attribute>
    <widget class="QPushButton" name="add_type">
     <property name="geometry">
      <rect>
       <x>193</x>
       <y>140</y>
       <width>92</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>添加类型</string>
     </property>
    </widget>
    <widget class="QPushButton" name="del_type">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>140</y>
       <width>92</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>删除类型</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>25</y>
       <width>211</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>25</y>
       <width>92</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>选择</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>25</y>
       <width>68</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>导出路径:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="theme_box">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>80</y>
       <width>68</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>主题选择:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>68</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>字体设置:</string>
     </property>
    </widget>
    <widget class="QFontComboBox" name="fontComboBox">
     <property name="geometry">
      <rect>
       <x>294</x>
       <y>80</y>
       <width>106</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>140</y>
       <width>68</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>消费类型:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="consume_box">
     <property name="geometry">
      <rect>
       <x>91</x>
       <y>140</y>
       <width>87</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="per_setting">
    <attribute name="title">
     <string>记录设置</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="bill_setting">
    <attribute name="title">
     <string>账单设置</string>
    </attribute>
    <widget class="QListView" name="bill_list_view">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>271</width>
       <height>411</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="add_bill">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>92</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>新建</string>
     </property>
    </widget>
    <widget class="QPushButton" name="del_bill">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>70</y>
       <width>92</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
